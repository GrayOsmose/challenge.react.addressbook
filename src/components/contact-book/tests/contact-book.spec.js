// Link.react-test.js
import React from 'react';
import { shallow } from 'enzyme';

import ContactBook from '../contact-book';

describe('ContactBook', function() {
  it('can render', function() {
    const component = shallow(<ContactBook />);
    expect(component).toMatchSnapshot();
  });
});
