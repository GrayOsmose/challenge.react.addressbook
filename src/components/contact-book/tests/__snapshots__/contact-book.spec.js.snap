// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactBook can render 1`] = `
<div
  className="col-sm-12"
>
  <div
    className="row"
  >
    <h1>
      Address-Book
    </h1>
    <Link
      replace={false}
      to="/new"
    >
      <Button
        active={false}
        block={false}
        bsClass="btn"
        bsStyle="default"
        disabled={false}
      >
        <span
          className="glyphicon glyphicon-plus"
        />
      </Button>
    </Link>
  </div>
  <div
    className="row"
  >
    <div
      className="col-md-4"
    >
      <Connect(ContactList) />
    </div>
    <div
      className="col-md-8"
    >
      <Route
        component={[Function]}
        exact={true}
        path="/"
      />
      <Route
        component={[Function]}
        path="/new"
      />
      <Route
        component={[Function]}
        path="/view/:contactId"
      />
      <Route
        component={[Function]}
        path="/edit/:contactId"
      />
    </div>
  </div>
</div>
`;
